README.md

```diff
@@ -50,6 +50,11 @@ NAME                      DESCRIPTION             VERSION(S)
 cluster.etcd.coreos.com   Managed etcd clusters   v1beta1
 ```
 
+When deleting etcd operator deployment, delete its leader election endpoint as well:
```

Comment:
i do not think this is a good place. it is not discoverable.

Probably it is time for us to create the initial op-guide with our first section:

- uninstall etcd operator

we mention how to delete the operator, how to remove the endpoint, and tell users they also need to manually remove all etcd clusters managed by etcd operator manually (or remove all the TPRs before delete the operator.).